config/routes.rb

```diff
@@ -3,4 +3,7 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/studios", to: "studios#index"
+  get "/movies/:id", to: "movies#show"
```

Comment:
You'll need a route here for submitting the form. A RESTful option could be:
`post "/movies/:id/actors", to: "actor_movies#create"`